processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -16,7 +16,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.naming;
+package org.mapstruct.ap.spi;
```

Comment:
I think that Hibernate Validator does something similar for their SPI default implementations. Anyway, I put in the ' hasser' as a ready-made alternative. That helps for my own testing as well and serves as example at the same moment.